### PR TITLE
Reset password pages

### PR DESCRIPTION
**Pull request by BenIraa**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,36 +5,48 @@
...
+    "autoprefixer": "^9.8.8",
     "babel-loader": "^8.2.4",
     "html-webpack-plugin": "^5.5.0",
     "jest": "^27.5.1",
     "jest-environment-jsdom": "^27.5.1",
+    "postcss": "^7.0.39",
     "prettier": "^2.6.2",
     "react-router-dom": "^6.3.0",
     "react-test-renderer": "^18.0.0",
-    "regenerator-runtime": "^0.13.9"
```
Comment (kareraolivier) [inline package.json:35]: this package is here twice

File: src/Components/Home.jsx [hunk 2]
```diff
@@ -1,9 +1,13 @@
-import React, { Component } from "react";
```
Comment (kareraolivier) [inline src/Components/Home.jsx:1]: what about component imported here?

**Review by kareraolivier — CHANGES_REQUESTED** [review]
I think those two things have to be changed

File: package.json (hunk 1 above)
Comment (BenIraa) [reply package.json:35]: I don't have it Local as twice

File: src/Components/Home.jsx (hunk 2 above)
Comment (BenIraa) [reply src/Components/Home.jsx:1]: same here!

**Comment by mansurissa** [comment]
You have confilicting files @BenIraa

File: src/Components/Home.jsx (hunk 2 above)
Comment (BenIraa) [reply src/Components/Home.jsx:1]: the contents of the red are the ones that have been changed unnecessarily not added

**Pull request closed without merging**